management-account/terraform/sso-admin-account-assignments.tf

```diff
@@ -259,7 +259,8 @@ locals {
       account_ids = [
         aws_organizations_account.modernisation_platform.id,
         aws_organizations_organization.default.master_account_id,
-        aws_organizations_account.organisation_security.id
+        aws_organizations_account.organisation_security.id,
+        aws_organizations_account.youth_justice_framework_management.id
```

Comment:
This is fine, but it's not `admin` permission set, but `modernisation_platform_engineer`.

Reply:
Thank you. Have moved to the correct block. Resubmitting.